src/components/DocumentationTopic/OnThisPageStickyContainer.vue

```diff
@@ -0,0 +1,60 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2021 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="OnThisPageStickyContainer" :class="{ hidden: isHidden }">
+    <slot />
+  </div>
+</template>
+
+<script>
+// 660px content + (170px aside + 22px padding-right)*2 + 28px*2 gutter
+export const ON_THIS_PAGE_CONTENT_BREAKPOINT = 1110;
+// 1080 content + (170px aside + 22px padding-right)
+export const ON_THIS_PAGE_CONTENT_BREAKPOINT_BIG = 1272;
+
+export default {
+  name: 'OnThisPageStickyContainer',
+  inject: ['store'],
+  computed: {
+    isHidden: ({ store }) => {
```

Comment:
I'm guessing you already looked into this, but was it not possible to do this with pure CSS? I'm noticing that sometimes the transition between showing/hiding this element can be not the smoothest (but most of the time it looks fine).

Reply:
Ok, just checking. I was right that you already looked into it :P